commit bfc8c1e32cdbd411215dcd03b9c126e38fb2a67b (refs/changes/97/1497/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-02-10T15:31:12-08:00 (2 years, 8 months ago)
    
    MANTA-3161 sdc-manta.git should run catest with the right node
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/Makefile b/Makefile
index 74b9d2f..316845a 100644
--- a/Makefile
+++ b/Makefile
@@ -92,7 +92,7 @@ check:: $(NODE_EXEC)
 
 .PHONY: test
 test: | $(CATEST)
-	$(CATEST) -a
+	PATH="$(TOP)/build/node/bin:$$PATH" $(CATEST) -a
 
 $(CATEST): deps/catest/.git
 
